dev/io.openliberty.microprofile.metrics.internal.3.0/src/io/openliberty/microprofile/metrics30/setup/config/DefaultBucketConfiguration.java

```diff
@@ -0,0 +1,53 @@
+package io.openliberty.microprofile.metrics30.setup.config;
```

Comment:
Need to check, what type of copyright we put on it. We still need copy right header i think. We've wrote this for small rye, but not sure if we use the SR copyright (it's in a separate file in the repo).

Reply:
Applies to all the classes here in the package.